src/scoped.component.js

```diff
@@ -78,7 +78,7 @@ export class Scoped extends React.Component {
 
       // The css to append to the dom
       const kremlingSelector = `[${kremlingAttrName}="${kremlingAttrValue}"]`;
-      const transformedCSS = props.css.replace(/& (.+){/g, (match, cssRule) => {
+      const transformedCSS = props.css.replace(/& ([^{}])+{/g, (match, cssRule) => {
```

Comment:
I think this should be fine, essentially we'll match anything except where a new rule starts with a bracket